### PR TITLE
Adding docs for onboardings and on demand Mendix scanning

### PR DESCRIPTION
**Pull request by MichielCuijpers**
@dessens0 could you review this PR for me please?

cc @dennis-sig

### CODE REVIEW

File: docs/mendix-teamserver-api.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+### 1. Trigger an QSM Onboarding workflow
+
+POST `/qsm/{customer}`
+
+#### Info
+Note that this endpoint requires a [SIGRID CI Authentication token](authentication-tokens.md) that is valid for {customer}. 
+##### Headers
+    - 'Authorization: Bearer YOUR_SIGRID_CI_Auth_TOKEN'
```
Comment (dessens0) [inline docs/mendix-teamserver-api.md:21]: Call the token 'YOUR_TOKEN' like in the example request (or call the token in the request 'YOUR_SIGRID_CI_Auth_TOKEN') so it's clear they are the same thing.

File: docs/mendix-teamserver-api.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Sigrid API for Mendix customers using the Teamserver
+
+This document describes 2 use cases to trigger Sigrid for customers that are using the central Mendix Teamserver.
+
+please note: When the customer is using the Bring Your Own Git scenario then the below does not apply. please refer to [Mendix for GitHub](mendix-github-actions.md)
+
+## Control the onboarding
+Onboarding a new Mendix system is normaly done via [addon.mendix.com](addon.mendix.com). In some cases customers would like to take onboarding in their own hands. Examples can be automation or bulk onboarding. In those cases customers can trigger an QSM Onboarding workflow that will add a new Mendix system to an existing Sigrid. By default the mainline of the Mendix system will be scanned once a day. The scan will provide both maintanability, OSH and security findings for QSM customers. AQM customers will only see maintainability results. The onboarding will generate a status email to the user specified in the payload.
```
Comment (dessens0) [inline docs/mendix-teamserver-api.md:8]: I think we can leave out the word 'workflow' throughout the document.

File: docs/mendix-teamserver-api.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    - appName : to be used as system name in Sigrid. You can use the Mendix project name, it will be converted to a Sigrid system name.
+
+    - mendixToken : we prefer the Mendix PAT [Personal Access Token](https://warden.mendix.com) MX model read only. There is support for the old API Key. In case of an API Key, the userName should match the key
+
+    - userName : to be used to send status email, not relevant for authentication in case of PAT
+    
+
+#### Example request
+```
```
Comment (dessens0) [inline docs/mendix-teamserver-api.md:36]: you add 'bash' here, so ```bash which may do some nice syntax highlighting

File: docs/mendix-teamserver-api.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+### 2. Trigger a QSM run workflow for an already onboarded system
+
+POST `/qsm/{customer}/{system}`
+
+#### Info
+Note that this endpoint requires a [SIGRID CI Authentication token](authentication-tokens.md) that is valid for {customer}.
+##### Headers
+    - 'Authorization: Bearer YOUR_SIGRID_CI_Auth_TOKEN'
```
Comment (dessens0) [inline docs/mendix-teamserver-api.md:48]: Same comment about the token

File: docs/mendix-teamserver-api.md (hunk 3 above)
Comment (dessens0) [reply docs/mendix-teamserver-api.md]: I committed the change.

**Review by dessens0 — APPROVED** [review]

File: docs/mendix-teamserver-api.md (hunk 3 above)
Comment (MichielCuijpers) [reply docs/mendix-teamserver-api.md]: thanks

**Pull request merged**